README.md

```diff
@@ -85,9 +76,11 @@ func main() {
 import "github.com/hellofresh/stats-go/bucket"
 
 timing := statsClient.BuildTimer().Start()
-operations := bucket.MetricOperation{"orders", "order", "create"}
+operation := bucket.MetricOperation{"orders", "order", "create"}
 err := orderService.Create(...)
-statsClient.TrackOperation("ordering", operations, timing, err == nil)
+statsClient.TrackOperation("ordering", operation, timing, err == nil)
```

Comment:
Not related to this PR, I think we should plan a refactoring soon for smth like `stats.TrackOperation(stats.Config{Name: ....})` in order to make it a bit more meaningful